src/test/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfigurationTest.java

```diff
@@ -0,0 +1,67 @@
+package com.atlassian.bitbucket.jenkins.internal.config;
```

Comment:
Can you please add tests for the new methods you added to `BitbucketPluginConfiguration`? And maybe the other uncovered methods while you're here?